tests/unit/driver/network/test_network_sync_driver.py

```diff
@@ -232,13 +294,12 @@ def _send_inputs_interact(cls, **kwargs):
 
 
 def test_send_configs(monkeypatch, sync_network_driver):
-    def _acquire_priv(cls, **kwargs):
+    def _acquire_appropriate_privilege_level(cls, **kwargs):
         return
 
-    # patching acquire priv so we know its called but dont have to worry about that actually
-    # trying to happen
     monkeypatch.setattr(
-        "scrapli.driver.network.sync_driver.NetworkDriver.acquire_priv", _acquire_priv
+        "scrapli.driver.network.sync_driver.NetworkDriver._acquire_appropriate_privilege_level",
```

Comment:
I *think* this is why ci is failing -- since we didn't modify send configs to use the new `_acquire_appropriate_privilege_level` its just calling "normal" `acquire_priv` which isn't patched so its raising that exception.